src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-form-control {
+    padding-top: $clr-forms-baseline * 4;
+
+    .clr-control-label {
+        display: block;
+        color: $clr-forms-label-color;
+        font-size: calc(#{$clr-forms-baseline} * 2 + 1px);
```

Comment:
@mathisscott created a mixin and a map where all font styles are added. More info here:
https://github.com/vmware/clarity/blob/master/src/clr-angular/button/_buttons.clarity.scss#L96
https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/_mixins.clarity.scss#L6
https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/_maps.clarity.scss#L568

Can we move this there? Not sure if we can because you are using variables here but just making a note to check if we can.

Reply:
The map makes sure that we are following the typography recommendations from UX.